flat_pack_library.scad

```diff
@@ -0,0 +1,87 @@
+//openscad flat-pack joint library
+
+
+
+module outsideCuts(length=100, finger=8, material=5, text=false, center=false, font="Liberation Sans") {
```

Comment:
I'm not sure I understand what the purpose of the defaults here is. Is there something special about 10cm, with 8mm fingers and 5mm thickness? These parameters will likely vary with application considerably.

Reply:
Also, perhaps a small comment could be placed above the module just mentioning what the parameters are. Is it the length of the fingers or the number of fingers? etc.

Reply:
The defaults ensure that something is generated similar to the 1x1x1 cube that a `cube();` with no parameters generates. Is this bad practice? The default values are nothing special.

Reply:
Ah, makes sense. In programming generally such defaults are bad, but if other parts of OpenSCAD do it then it follows to do it here too. Are there values that feels more like 1, 1, 1 where still renders something, but feels less arbitrary?

Reply:
I'll look into that; it's possible that this is a bad idea.